tock/tock/static/js/components/notifications.js

```diff
@@ -1,52 +1,79 @@
 (function setupNotifications() {
-  if("Notification" in window) {
-    Notification.requestPermission().then(function(result) {
-      // If the user grants permission, or has granted permission in the past,
-      // setup the timers.  Otherwise bail out.
-      if (result !== 'granted') {
-        return;
+
+  var iconPath = (function() {
+    var links = document.getElementsByTagName('link');
+    var paths = [].slice.call(links).map(function (e) {
+      if (e.getAttribute('rel') === 'shortcut icon') {
+        return e.getAttribute('href');
```

Comment:
Instead of iterating through all links, will this work?
```
iconPath = document.querySelector("[rel='shortcut icon']").href;
```
You'd still want to be sure that iconPath was found

Reply:
thanks for the suggestion. I submitted an updated patch.